package.json

```diff
@@ -83,6 +89,7 @@
     "lodash": "^4.17.20",
     "lottie-ios": "3.1.8",
     "lottie-react-native": "^3.5.0",
+    "protobufjs": "^6.10.2",
```

Comment:
Did we still need this one even with `@helium/proto-ble`?

Reply:
yeah, I use one of their utilities to do some decoding in `src/utils/bluetooth/bluetoothDataParser.ts`